src/frontend/flowr-window.ts

```diff
@@ -7,6 +7,10 @@ export class FlowrWindow extends BrowserWindow {
   private resizeTimeout?: number
   private player: Player
 
+  get phoneServerUrl() {
+    return this.store.get('phoneServer')
```

Comment:
I suggest to have a share store between PhoneWindows and FlowrWindows rather that having the store as a private member of FlowrWindows. The Idea is to avoid exposing phoneServerUrl form FlowrWindows because it is not its matter (scope).